src/api/app/views/webui2/webui/repositories/_repository_entry.html.haml

```diff
@@ -0,0 +1,7 @@
+.col-12.col-md-6.col-lg-4
+  .card.mb-3.repository-card
+    .card-header
+      = link_to(repository, project_repository_state_path(project: project, repository: repository), class: 'font-weight-bold')
+      .small
+        #{repository.architectures.pluck(:name).join(', ')}
```

Comment:
When no architectures are selected, nothing is displayed. Comparing with the old UI, we displayed back then `()`. Now, I believe we should display something along the lines of `No architecture selected`